### ISSUE TITLE
Please upload missing files

### ISSUE THREAD

**Issue by wwwMADwww** [ExCellRaD/osuElements#11]
Hello! Please upload missing project "TestApp" and files for project "TestContent".

**Comment by JasperDeSutter** [comment]
Yeah for testing this was all just some quick and dirty work, so I'd rather not include it in the repository.

If you really need this you can add the files from [asphyxia's united](https://osu.ppy.sh/beatmapsets/237768/#osu/550235) right in the TestContent project
(best to use the osu client and then copy the beatmap folder straight from your osu!/Songs folder)

If you comment out/remove the unnecessary code, feel free to send a PR so the repository stays a bit more clean.
